src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,31 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static IDataTransferManagerInterop dataTransferManagerInterop = DataTransferManager.As<IDataTransferManagerInterop>();

```

Comment:
This should be readonly, no?

Reply:
![image](https://github.com/microsoft/CsWinRT/assets/49179966/1432fa12-0d73-4239-bb7d-a772dcd92bc3)

Currently, other similar interop apis do not have the readonly keyword, and if you need to add the readonly keyword, you may need to create a new PR and change all the interop apis